Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -45,15 +52,22 @@ protected class HandPanData
         [SerializeField]
         private bool lockVertical = false;
         [SerializeField]
-        private bool wrapTexture = false;
+        [Tooltip("If this is checked, Max Pan Horizontal and Max Pan Vertical are ignored.")]
+        private bool unlimitedPan = true;
+        [SerializeField]
+        [Range(1.0f, 20.0f)]
+        private float maxPanHorizontal = 2;
         [SerializeField]
-        private bool velocityActive = false;
+        [Range(1.0f, 20.0f)]
+        private float maxPanVertical = 2;
```

Comment:
@johnppella @keveleigh are these units something that are well understood by most Unity developers (i.e. am I just ignorant because I'm new) or do they have some special meaning (i.e. why default is 2, what's the significance of 20).

Reply:
these are velocities that experimentally represent reasonable bounds.

Reply:
Sounds good, whenever I see numbers without units I always have to ask the same question :)